### DISCUSSION TITLE
Storing InDesign Work in Pimcore

### DISCUSSION THREAD

**Discussion by cancan101** [pimcore/pimcore#17500] [Q&A / Support]
What is the best way to store InDesign content in Pimcore? The naive approach would be to ["Package" the InDesign work](https://creativecloud.adobe.com/learn/indesign/web/save-share-projects) which creates a folder with `indd`, `indml` as well as folders of `Document fonts` and `Links` and then to upload that folder to Pimcore. One issue that arrises is that Pimcore might rename the file in the links folder (ie if there are naming issues) which then will cause broken links when downloading. 

Is there a better way to handle this?

**Comment by developkariyer** [comment]
I am uploading two files:
- PDF or any Pimcore-thumbnail capable cover file
- A zip file containing all content

Normally, zip should be enough but I also need to see what is inside zip without downloading, so other cover file covers this :)

Also, I am manipulating file names and folders after upload or attached to DataObjects, in order to have a tidy workspace, since I clone Assets  folder to Google Drive also. This is somehow dirty but it works for me.

Regards.
Umut.
